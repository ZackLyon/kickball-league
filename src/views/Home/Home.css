.home-page {
  width: 100vw;
  height: 100vh;
  background-size: cover;
  background-repeat: none;
  display: flex;
  justify-content: center;
}

.kickball-title {
  font-size: 5rem;
  font-weight: bold;
  position: relative;
  top: 20vh;
  color: rgb(115, 255, 176);
  text-shadow: 3px 3px rgb(212, 58, 38);
}

.acknowledgement {
  font-weight: bold;
  position: absolute;
  bottom: 1vh;
  left: 1vw;
  padding: 0.5vw;
  background-color: rgba(255, 255, 255, 0.5);
}
