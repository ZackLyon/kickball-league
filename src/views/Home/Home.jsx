import React from 'react';
import './Home.css';

export default function Home() {
  return (
    <div style={{ backgroundImage: 'url(kickball.jpg)' }} className='home-page'>
      <div className='kickball-title'>PNW Kickball League</div>
      <div className='acknowledgement'>Photo by cottonbro from Pexels</div>
    </div>
  );
}
