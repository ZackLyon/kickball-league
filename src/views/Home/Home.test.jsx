import { render, screen } from '@testing-library/react';
import Home from './Home';

it('should render the Home view', () => {
  const { container } = render(<Home />);

  expect(container).toMatchSnapshot();
});
