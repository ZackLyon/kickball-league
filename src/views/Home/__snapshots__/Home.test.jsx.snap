// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Home view 1`] = `
<div>
  <div
    class="home-page"
    style="background-image: url(kickball.jpg);"
  >
    <div
      class="kickball-title"
    >
      PNW Kickball League
    </div>
    <div
      class="acknowledgement"
    >
      Photo by cottonbro from Pexels
    </div>
  </div>
</div>
`;
